Example/Artsy-UIButtons.xcodeproj/project.pbxproj

```diff
@@ -389,7 +389,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
```

Comment:
Yeah you can revert this. In fact, unless you need to make any dependency changes, you don’t need to run `pod install` at all anymore, provided you do checkout the submodule. If you haven’t done so yet:

```
$ cd Example
$ rm -rf Pods
$ git reset HEAD Pods
$ cd ..
$ git submodule update --init
```

This is from the top of my head. If it doesn’t work, maybe just make a new clone of the repo and run the last command and apply your code change there.